packages/frontend/core/src/blocksuite/presets/ai/_common/chat-actions-handle.ts

```diff
@@ -158,6 +161,7 @@ function addAIChatBlock(
     {
       xywh: bound.serialize(),
       messages: JSON.stringify(messages),
+      index: layer.generateIndex('affine:embed-ai-chat'),
```

Comment:
If you need to add block at the very top, you could change the parameter to 'shape'.